optaplanner-core/src/main/java/org/optaplanner/core/api/domain/constraintweight/ConstraintConfiguration.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.core.api.domain.constraintweight;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.*;
+import static java.lang.annotation.RetentionPolicy.*;
+
+/**
+ * Specifies that the class holds a number {@link ConstraintWeight} annotated members.
```

Comment:
```suggestion
 * Specifies that the class holds a number of {@link ConstraintWeight} annotated members.
```

Reply:
+1, these suggestions are intresting - I would have applied them directly but I have local changes that I fear might conflict later. Next time, I 'll make sure to look a the Pr review first to merge any of these.